lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -642,8 +642,10 @@ void LocalEnforcer::complete_final_unit_action_flows_install(
       return;
     }
     case RESTRICT_ACCESS: {
+      // Don't send IP to allow session retrieval from session store while
+      // aggregating records.
       pipelined_client_->add_gy_final_action_flow(
-          imsi, ip, final_action_info.restrict_rules, {});
```

Comment:
lets hold this. I am doing some changes that will have some impact on this

Reply:
I just remember that we do need the IP for pipelined to find the flow. Lets remove this change

Reply:
This is not needed anymore after fixing session retrieval in SessionD (PR 2960).